src/Controller/CaseEventController.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\CaseEntity;
+use App\Entity\CaseEvent;
+use App\Entity\CaseEventPartyRelation;
+use App\Entity\Party;
+use App\Form\CaseEventEditType;
+use App\Form\CaseEventFilterType;
+use App\Form\CaseEventNewType;
+use App\Repository\CaseEventRepository;
+use App\Service\PartyHelper;
+use DateTime;
+use Doctrine\Common\Collections\Criteria;
+use Doctrine\ORM\EntityManagerInterface;
+use Knp\Component\Pager\PaginatorInterface;
+use Lexik\Bundle\FormFilterBundle\Filter\FilterBuilderUpdaterInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\Translation\TranslatableMessage;
+
+#[Route('/case/{id}/case-events')]
+class CaseEventController extends AbstractController
+{
+    #[Route('/', name: 'case_event_index', methods: ['GET'])]
+    public function index(CaseEntity $case, CaseEventRepository $caseEventRepository, FilterBuilderUpdaterInterface $filterBuilderUpdater, PaginatorInterface $paginator, Request $request): Response
+    {
+        $filterOptions = [
+            'case' => $case,
+            'method' => 'get',
+            'action' => $this->generateUrl('case_event_index', ['id' => $case->getId()]),
+        ];
+
+        $filterForm = $this->createForm(CaseEventFilterType::class, null, $filterOptions);
+        if ($request->query->has($filterForm->getName())) {
+            $filterForm->submit($request->query->get($filterForm->getName()));
+        }
+
+        $filterBuilder = $caseEventRepository->createAvailableCaseEventsForCaseQueryBuilder('ce', $case);
+        $filterBuilderUpdater->addFilterConditions($filterForm, $filterBuilder);
+        $query = $filterBuilder->getQuery();
+
+        $pagination = $paginator->paginate(
+            $query,
+            1,
+            1000, // Hopefully this is never reached.
```

Comment:
Why do we use a paginator if we do not want to paginate?

Reply:
Thats a good point.. Fixed.